app/src/main/java/app/revanced/manager/ui/component/bundle/BundleSelector.kt

```diff
@@ -54,6 +57,12 @@ fun BundleSelector(bundles: List<PatchBundleSource>, onFinish: (PatchBundleSourc
             }
             bundles.forEach {
                 val name by it.nameState
+                var version by remember { mutableStateOf<String?>(null) }
+
+                LaunchedEffect(Unit) {
+                    version = it.currentVersion()
+                }
```

Comment:
Use the props flow instead. There should be an example for getting the version in the bundle information page.